aten/src/TH/generic/THTensorApply.hpp

```diff
@@ -8,6 +8,7 @@
 #define ORDIN_TH_OMP_OVERHEAD_THRESHOLD 20000
 #define UNCERTAIN_TH_OMP_OVERHEAD_THRESHOLD 50000
 #define TH_OMP_OVERHEAD_THRESHOLD 100000
+#define TH_PARALLEL_OVERHEAD_THRESHOLD 80000
```

Comment:
Please move it into the THTensorEvenMoreMath.cpp as constexpr, we are trying to get rid of THTensorApply.
Also something like `parallel_overhead_threshold_index_select` would be better as name ( + comment how we selected this `80000` value)